src/web/templates/admin/screens/screen_card.html

```diff
@@ -16,46 +16,26 @@
                     <span class="pe-1">{{ macros.screen_type_icon(screen.type, true) }}</span>
                     {{ _('Type: %(screen_type)s', screen_type=screen.type_str) }}
                 </div>
-                <div>
-                    {{ _('Columns:') }} {% for i in range(screen.columns) %}<i class="bi-square-fill"></i> {% endfor %}
-                </div>
-                <div class="{% if not screen.menu_link %}text-secondary{% endif %}">
-                    <i class="bi-hand-index pe-1"></i>
-                    {% if screen.menu_link %}
-                        {{ _('Menu link label: %(menu_label)s', menu_label=screen.menu_label) }}
-                        {% if not screen.menu_text %}
-                            <span class="fst-italic">
-                                {{ _('(by default)') }}
-                            </span>
-                        {% endif %}
-                    {% else %}
-                        <span class="fst-italic">
-                            {{ _('Menu link label: none') }}
-                        </span>
-                    {% endif %}
-                </div>
-                <div class="{% if not screen.menu %}text-secondary{% endif %}">
-                    <i class="bi-three-dots pe-1"></i>
-                    {% if screen.menu %}{{ _('Menu: %(menu)s', menu=screen.menu) }}{% else %}<span class="fst-italic">{{ _('Menu: none') }}</span>{% endif %}
-                </div>
-                <div class="{% if not screen.timer %}text-secondary{% endif %}">
-                    <i class="bi-clock-fill pe-1"></i>
-                    {% if screen.timer %}{{ _('Timer: %(timer_uniq_id)s', timer_uniq_id=screen.timer.uniq_id) }}{% else %}<span class="fst-italic">{{ _('Timer: none') }}</span>{% endif %}
-                </div>
             {% endif %}
-            {% if screen.type == 'input' %}
-                <div>
-                    <i class="bi-x-circle pe-1"></i>
-                    {{ _('Exit button:') }}
-                    {% if screen.input_exit_button %}
-                        {{ _('yes') }}
+            {% if screen.type == 'ranking' %}
```

Comment:
I can't see a difference between cards for ranking or crosstable, can we show this?